### ISSUE TITLE
Parse Markdown Strings to Json Object

### ISSUE THREAD

**Issue by yonderbread** [klaytonfaria/markdown-json#16]
Instead of reading in files, would I be able to parse some strings with this and convert them to objects?

**Comment by klaytonfaria** [comment]
@yonderbread , thanks for the suggestion. Could you please give more details about the string input usage?

**Comment by yonderbread** [comment]
Yea of course, so instead of reading directly from a file object you could just pass an already read out file as a string to markdown-json and it would return the Object version of it.

**Comment by johnbeech** [comment]
👍  from me - I wasn't looking for a server/file processor - I just wanted a function that would convert a markdown string to a JSON structure.

e.g.:
```js
const markdownJson = require('markdown-json')
const fs = require('fs')
const markdown = fs.readFileSync('README.md', 'utf8')
const json = markdownJson.parse(markdown)
console.log(json)
```

**Comment by yonderbread** [comment]
> Same sentiment from me - I wasn't looking for a server/file processor - I just wanted a function that would convert a markdown string to a JSON structure.
> 
> e.g.:
> 
> ```js
> const markdownJson = require('markdown-json')
> const fs = require('fs')
> const markdown = fs.readFileSync('README.md', 'utf8')
> const json = markdownJson.parse(markdown)
> console.log(json)
> ```

Yup. That's what I remember wanting to do as well. If I find some time I will try to implement this into the library.

**Comment by uarefans** [comment]
> 

maybe this is solution to not convert markdown to html

1. Open lib/generator.js
2. Edit line 125
3. Give comment 
`// .use(markdown())`
4. Save and run again

**Comment by yonderbread** [comment]
Gonna try to [test some things out.](https://github.com/klaytonfaria/markdown-json/blob/140d332c4766bf0da50b99facb5ee82abff4c975/lib/generator.js#L125)

**Comment by uarefans** [comment]
> Gonna try to [test some things out.](https://github.com/yonderbread/markdown-json/blob/d01f3257739a53ba625c21af77246a3b7a75a990/lib/generator.js#L125) Feel free to open a pull request on my fork.

not
```
//.use(writeJson)
```
but
```
 // .use(markdown())
```
if i reference your file at line 123
and please test delete all generator.js and run node markdown-json again, you give error or not. This is important because if not show any error you are using global markdown-json not from github but from **npm install markdown-json**. 

reinstall with 
```bash
git clone https://github.com/klaytonfaria/markdown-json.git
cd markdown-json
npm install
edit lib/generator.js
node markdown-json
```
